README.md

```diff
@@ -1,3 +1,31 @@
+* *Status:* Production
```

Comment:
Confirmation gotten! Headers that aren't at the top are OK, she says. You're cool, and I'll move mine.